### PR TITLE
New: Tag prefixes and default enforcer behavior for namespaces

### PR DESCRIPTION
**Pull request by ericrpowers**
#### Description
To narrow down suggestions and help make things a bit more lightweight, we want to add a list of tag prefixes that will be used to suggest policies. Only these tags will be transmitted on the wire.

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "937",
    "commit-sha": "67341789017016dbe86a5e6cf66a9f1a6ad71a9b",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "577",
    "commit-sha": "a9d17fcce4e45ab971616c870626878c0f4498c5",
    "pipeline": "master"
  }
]
```

**Review by t00f — DISMISSED** [review]
Lgtm

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "937",
    "commit-sha": "ab32c457bffc3a56290f4ade9e8cd0fe0b83ae6e",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "577",
    "commit-sha": "a9d17fcce4e45ab971616c870626878c0f4498c5",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
@dstiliadis @primalmotion @mheese please have a look - will merge this PR by EOD otherwise

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "937",
    "commit-sha": "4515ac331f76f7a84bab8f676def81216284950e",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "577",
    "commit-sha": "a9d17fcce4e45ab971616c870626878c0f4498c5",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "577",
    "commit-sha": "9c1ffdb1e032ebef5190bec53db3d6a42871f7bb",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "937",
    "commit-sha": "26b5f8451255f237648875b0ab6540f31cf0a221",
    "pipeline": "master"
  }
]
```

File: specs/namespacepolicyinfo.spec [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  description: Returns the policy info of the specified namespace.
+
+# Attributes
+attributes:
+  v1:
+  - name: behavior
+    description: The default enforcer behavior for the namespace.
+    type: enum
+    exposed: true
+    stored: true
```
Comment (primalmotion) [inline specs/namespacepolicyinfo.spec:17]: These props are not stored and are read only,

File: specs/root.spec [hunk 2]
```diff
@@ -646,6 +646,12 @@ relations:
   create:
     description: Creates a new namespace mapping.
 
+- rest_name: namespacepolicyinfo
+  get:
+    description: Returns the policy info of the specified namespace.
+  create:
+    description: Modifies the policy info of the specified namespace.
```
Comment (primalmotion) [inline specs/root.spec:653]: Nope. Only get here. You must have access to the real namespace

**Review by primalmotion — CHANGES_REQUESTED** [review]

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "937",
    "commit-sha": "e476a7e26d7602460ebfd18a892a4e9b66b83a99",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "577",
    "commit-sha": "8c23c8a77bd08f991b9ff0a134b209d496ad6bd1",
    "pipeline": "master"
  }
]
```

**Review by primalmotion — APPROVED** [review]

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "tag-prefixes",
    "component": "backend",
    "pr-id": "937",
    "commit-sha": "447dead73661c0f6ed7ac8c7c127c00910515017",
    "pipeline": "master"
  },
  {
    "project": "tag-prefixes",
    "component": "gaia",
    "pr-id": "577",
    "commit-sha": "8c23c8a77bd08f991b9ff0a134b209d496ad6bd1",
    "pipeline": "master"
  }
]
```

**Pull request merged**